md/monitoring.md

```diff
@@ -1,32 +1,35 @@
-# Process monitoring
+# Monitor open cases
 
-Monitoring gives information relative to a process ongoing execution.
+This page describes the "Monitor open cases" page of the Administration profile in Bonita Portal and in the [Bonita Administrator Application](admin-application-overview.md).  
+In this page, _Administrators_ and _Process Managers_ can have a complete view of the sanity of the BPM operations, filter by process, access the process visualization and case details of a process. The page is available in both Bonita Portal and
```

Comment:
the end of the sentence is missing but is this last sentence necessary as it is already explained in the first sentence ?

Reply:
removing it